public/assets/images/icons/projectV2/PointMarkerIconsSymbol.tsx

```diff
@@ -0,0 +1,272 @@
+// Todo - move to a common location
+import type { IconProps } from '../../../../../src/features/common/types/common';
+
+export const Agroforestry = ({
+  color = 'defaultColor',
```

Comment:
I don't think `defaultColor` does anything - you'd need to define a css variable for this to work.

A simpler approach would be to simply define a default color of #333, or use `currentColor` instead.